Build/Build.cs

```diff
@@ -288,10 +288,10 @@ class Build : NukeBuild
     string YarnCli => $"{ToolPathResolver.GetPackageExecutable("Yarn.MSBuild", "yarn.js", "1.22.19")} --silent";
     
     bool HasDocumentationChanges =>
-        Changes.Any(x => x.StartsWith("docs"));
+        Changes.Any(x => x.StartsWith("docs", StringComparison.InvariantCultureIgnoreCase));
```

Comment:
🤔 I don't think this is very important. It's just a build script and we don't have culture-sensitive folders.